app/controllers/devise/registrations_controller.rb

```diff
@@ -165,4 +177,11 @@ def sign_in_after_change_password?
 
     Devise.sign_in_after_change_password
   end
+
+  def sign_out_resource(resource_name, resource)
+    Devise.sign_out_all_scopes ? sign_out : sign_out(resource_name)
+    set_flash_message! :notice, :destroyed
+    yield resource if block_given?
```

Comment:
A spec must have failed for that.

If not, it is missing.

We need to pass the block from destroy to this method.

Reply:
We didn't have any tests for the `destroy action` but I'm having trouble to check the success case for this implementation :thinking: